.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: cpp
+dist: bionic
+compiler: gcc
+
+script:
+- mkdir -p build
+- cd build
+- cmake -DCMAKE_BUILD_TYPE=Debug ..
+- make CXXFLAGS="-j2 -Wall"
```

Comment:
I doubt `CXXFLAGS` will be honoured here, you should set it at cmake configuration step instead.